### PR TITLE
Add more registration options

### PR DESCRIPTION
**Pull request by marcelo140**
This PR adds two new columns to an activity:

- **external_link**: allows the admins to specify an external registration platform in case they need to use a more complex process (ex: doodle)

- **allows_registrations**: the admins can now specify if the users can start signing up for the activity or not. This allows the admins to announce an event earlier even if they still don't know exactly when will it happen.

### CODE REVIEW

File: app/views/activities/_registration.slim [hunk 1]
```diff
@@ -1,10 +1,17 @@
-- if @activity.end_date.future?
-  - if can?(:destroy, Registration) && @activity.registered?(current_user)
+- registration = @activity.registrations.build
+
+- if @activity.external_link.empty?
+  - if can?(:destroy, registration) && @activity.registered?(current_user)
```
Comment (9thScientist) [inline app/views/activities/_registration.slim]: This condition should probably be a helper method.

File: app/views/activities/_registration.slim [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
       class: "btn btn-disabled btn-lg" do
       i.fa.fa-times.fa-fw
       | Anular Inscrição
-  - elsif can?(:create, Registration)
+  - elsif can?(:create, registration)
     = link_to "Inscrever-me!",
       activity_registration_path(@activity) || "#",
-      class: "btn border-theme btn-lg" + (@activity.end_date.past? ? " disabled": "")
+      class: "btn border-theme btn-lg"
+- elsif can?(:create, registration)
```
Comment (9thScientist) [inline app/views/activities/_registration.slim]: @fribmendes will probably want this in a helper method as well.

File: db/schema.rb [hunk 3]
```diff
@@ -31,9 +31,11 @@
     t.integer  "poster_file_size"
     t.datetime "poster_updated_at"
     t.integer  "activity_id"
-    t.datetime "created_at",                                             null: false
-    t.datetime "updated_at",                                             null: false
+    t.datetime "created_at",                                                             null: false
+    t.datetime "updated_at",                                                             null: false
```
Comment (9thScientist) [inline db/schema.rb]: Nothing wrong here, but why did you change the indentation?

Comment (marcelo140) [reply db/schema.rb]: It was generated
